docs/ref/contrib/admin/index.txt

```diff
@@ -1326,15 +1326,18 @@ templates used by the :class:`ModelAdmin` views:
 
 .. warning::
 
-    :meth:`ModelAdmin.save_model` and :meth:`ModelAdmin.delete_model` must
-    save/delete the object, they are not for veto purposes, rather they allow
-    you to perform extra operations.
+    When implementing :meth:`ModelAdmin.save_model` and
```

Comment:
LOL - so you're agreeing with my use of "overriding". I think I need more coffee. Yes. Let's use overriding. I'll update the PR shortly.